src/chromap.cc

```diff
@@ -150,8 +168,19 @@ void Chromap::TrimAdapterForPairedEndRead(uint32_t pair_index,
         // Trim adapters and TODO: fix sequencing errors
         int overlap_length =
             read2_length - seed_start_position + si * seed_length;
-        read_batch1.TrimSequenceAt(pair_index, overlap_length);
-        read_batch2.TrimSequenceAt(pair_index, overlap_length);
+        // The case that read1 is contained in read2
```

Comment:
Can we add more comment here on how this the contained case works? If I understand correctly, when a read is contained in another, no trimming will be performed? And the condition is when the length after trimming is equal to the sequence length?